### ISSUE TITLE
so, you just used the training set(1020 images) to fine tuned the AlexNet and got a good result?

### ISSUE THREAD

**Issue by huyangc** [jimgoo/caffe-oxford102#4]
I fine tuned the AlexNet with the training set, and the loss divergent, but when using the test set(6k images) instead, it converged. So I'm really curious about how you fine tuned the AlexNet and got a good result. : )

**Comment by huyangc** [comment]
However I don't know why but when I trained the net for the third time and it converged. However it is strange that the input form influence the result so much, raw image(jpeg) led to 78% precision, lmdb led to 86% precision. I can't get the 93% precision and I don't know why for now. maybe I can preprocess the input image such as flipping and so on to increase the numbers of input images to get a better precision.
